### PR TITLE
Tidying up from ./configure --with-adobe Mac link issue.  Removed website/ 

### PR DESCRIPTION
**Pull request by clanmills**

### CODE REVIEW

**Comment by clanmills** [comment]
This is mostly clean up work.

I've added a section 4.9 to WORK-IN-PROGESS about CMake/EXIV2_ENABLE_ADOBE_SDK  Let me know if I need to clarify anything about using the Adobe XMPsdk in v0.26.1

File: WORK-IN-PROGRESS [hunk 1]
```diff
@@ -474,11 +476,75 @@ Modified: 2017-08-18
...
+4.9 Building and linking XMPsdk with libexiv2
+
+    Exiv2 v0.26 (and many earlier editions) contain Adobe XMPsdk code.
+    The precise version of this code is unknown.
+    The code is stored in the xmpsdk/ directory which contains src/ and include/ directories.
+    The xmpsdk has it's own Makefile, CMakeLists.txt and msvc/xmpsdk/project files.
+    
+    Exiv2 v0.26 builds a static library libxmpsdk.a which is linked into libexiv2.a
+    
+    None of this is the change in Exiv2 v0.26.1
```
Comment (piponazo) [inline WORK-IN-PROGRESS]: Maybe rephrase to : None of this **has changed** in Exiv2 v0.26.1. ?

File: WORK-IN-PROGRESS [hunk 2]
```diff
@@ -474,11 +476,75 @@ Modified: 2017-08-18
...
+    
+    Exiv2 v0.26.1 will support a new option to download and build "pure Adobe" XMPsdk.
```
Comment (piponazo) [inline WORK-IN-PROGRESS]: We could make explicit here that this option is for the moment only available with **the autotools** and that we will work in other ways to bring those dependencies in the future.

**Review by piponazo — APPROVED** [review]
bye bye 24K lines of code :dagger:

File: WORK-IN-PROGRESS (hunk 1 above)
Comment (clanmills) [inline WORK-IN-PROGRESS]: Yup, that's a typo.

File: WORK-IN-PROGRESS (hunk 2 above)
Comment (clanmills) [inline WORK-IN-PROGRESS]: Yes.  That's possible.  Adobe XMPsdk is supported on Cygwin (although I'm confident of fixing that).  The user who requested this needs Adobe XMPsdk and msvc, so I'll move in that direction.  CMake/--adobe and CMake/Cygwin are not essential to v0.26.1.  I've also realised that there isn't enough time to get v0.26.1 to GM by the end of September.  However, we'll have v0.26.1 RC1 and request feedback.

**Comment by clanmills** [comment]
We can't remove xmpsdk/src and xmpsdk/include directories.  To retain backward compatibility, they will remain for all v0.26.x releases (and possibly longer).

**Comment by piponazo** [comment]
Yes, with my previous comment I was referring to the **website**  folder :smiley:

File: WORK-IN-PROGRESS (hunk 1 above)
Comment (clanmills) [inline WORK-IN-PROGRESS]: It's trivial to fix this later.  However, it's more interesting to fix it now.  I'm committed the fix to git on my machine.  I'll push to master and see what happens.

**Comment by clanmills** [comment]
Wow.  That's awesome.  Travis kicked in, the edited code has been merged.  That is awesome.  I'm very impressed.
